src/views/Message/index.js

```diff
@@ -20,7 +20,8 @@ export default function Message(props) {
         <div
           className={['bubble', `${isMine ? 'mine' : ''}`].join(' ')}
           title={friendlyTimestamp}>
-          {data.msg}
+          {/* {data.msg} */}
+          <div dangerouslySetInnerHTML={{ __html: data.msg }} />
```

Comment:
Why dangerouslySetInnerHTML?

Reply:
Links found in the message are replaced by textual anchor tag to maintain the way we store messages on the database (string format). dangerouslySetInnerHTML is used for rendering only the markup part of the message.